run/django/requirements.txt

```
@@ -1,4 +1,5 @@
-Django==4.0.4
+Django==4.0.4; python_version >= 3.8
+Django==3.2.13; python_version < 3.8
```

`python_version` needs to be a string to parse, according to [PEP-496](https://peps.python.org/pep-0496/) 

Some tests 'succeeding' because they're finding no tests. Example output: 

```
 Using noxfile-template from parent folder (/workspace).

nox > Running session py-3.8
nox > Creating virtual environment (virtualenv) using python3.8 in .nox/py-3-8
nox > Session py-3.8 was successful.
No tests found, skipping directory.
PWD: /workspace/appengine/standard_python3/django
```

```suggestion
Django==4.0.4; python_version >= '3.8'
Django==3.2.13; python_version < '3.8'
```